docs/pages/docs/tile/select-tile-group.mdx

```diff
@@ -68,10 +72,92 @@ FSelectTileGroup<Value>(
 );
 ```
 
+### `FTileGroup.builder(...)`
+
+```dart
+FSelectTileGroup<Value>.builder(
+  groupController: FMultiSelectGroupController(), // or FRadioSelectGroupController()
+  scrollController: ScrollController(),
```

Comment:
I had a random thought, but do you think it's a good idea to init `ScrollController` directly here? This pattern should technically never be used since we need to dispose the controller.

No strong opinions on it though.

Reply:
We mainly do it inside here instead of inside a stateful widget to avoid verbosity. That said, no strong opinions either.